### DISCUSSION TITLE
"Optech Make Me Famous!" label

### DISCUSSION THREAD

**Discussion by Kixunil** [rust-bitcoin/rust-bitcoin#1673] [General]
This label is intended for PRs that are considered so interesting that wider technical community should know about them. As the name suggests, the folks from bitcoin optech use the label to determine which changes are interesting enough to add to newsletter. Please use with care.

**Comment by apoelstra** [comment]
concept ACK

**Comment by tcharding** [comment]
Lol, I love the label title, my ego goes nuts when they link to a PR I did :)
